axelor-production/src/main/java/com/axelor/apps/production/service/manuforder/ManufOrderWorkflowServiceImpl.java

```diff
@@ -725,4 +725,49 @@ public void createPurchaseOrder(ManufOrder manufOrder) throws AxelorException {
 
     manufOrderRepo.save(manufOrder);
   }
+
+  @Override
+  @Transactional(rollbackOn = {Exception.class})
+  public void setOperationOrderMaxPriority(ManufOrder manufOrder) {
+
+    if (manufOrder == null
+        || Boolean.FALSE.equals(manufOrder.getProdProcess().getOperationContinuity())) {
+      return;
+    }
+
+    manufOrder = manufOrderRepo.find(manufOrder.getId());
+
+    List<OperationOrder> operationOrderList = manufOrder.getOperationOrderList();
+    int optionalOperationOrderLargestPriority =
+        operationOrderList.stream()
+            .filter(
+                order ->
+                    !order.getStatusSelect().equals(ManufOrderRepository.STATUS_FINISHED)
+                        && !order.getStatusSelect().equals(ManufOrderRepository.STATUS_CANCELED))
+            .filter(
```

Comment:
Please group all filters in one filter

Reply:
Here it was done in separate filter conditions to group similar conditions and make it readable
Can you please explain the reason why we need to group into one?